sdk/monitor/azquery/cloud_config.go

```diff
@@ -0,0 +1,41 @@
+//go:build go1.18
+// +build go1.18
+
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+package azquery
+
+import "github.com/Azure/azure-sdk-for-go/sdk/azcore/cloud"
+
+const (
+	ServiceNameLogs    cloud.ServiceName = "azqueryLogs"
```

Comment:
Missing doc comments.

Reply:
added in commit [be375c6](https://github.com/Azure/azure-sdk-for-go/pull/19462/commits/be375c658075097c3a95c800c5a584376381d5f4)